include/upgrader/prereq.inc.php

```diff
@@ -15,7 +15,7 @@
             <?php echo __('These items are necessary in order to run the latest version of osTicket.');?>
             <ul class="progress">
                 <li class="<?php echo $upgrader->check_php()?'yes':'no'; ?>">
-                <?php echo sprintf(__('%s or later'), 'PHP v5.4'); ?> - (<small><b><?php echo PHP_VERSION; ?></b></small>)</li>
+                <?php echo sprintf(__('%s or later'), 'PHP v5.6'); ?> - (<small><b><?php echo PHP_VERSION; ?></b></small>)</li>
```

Comment:
Use -  something like `SetupWizard::$prereq['php']` so we will have one spot to change it going forward. Better yet adding static methods might be better so you can make $prereq protected -> `protected $prereq = ....`

```php

static function getPHPVersion() {
     return self::$prereq['php']; 
}

```